### ISSUE TITLE
IOC: Stress rig reset after panic stop

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#4966]
As a user after I have pressed "Panic Stop" I would like to be able to make the stress rig usable again without having to remember to send it the arbitrary command C23,1. 

### Acceptance Criteria

1. Decision is made with scientists of when and how the C23,1 command is sent
1. One of
    1. Either: Button exists to send this command and it is clear that it needs pushing
    1. Or: Restore is sent on button release

### Notes

User says:

>‘Panic stop’ is rarely used so there’s no real urgency here, especially as now we know we can do a C23,1 to restore things. Ideally the Panic Stop button would switch off the hydraulics but not leave the hydraulic system in a disabled state. I guess this could be done with an immediate re-enable command, but a button that stayed in until clicked again (or the mouse released) would at least give the user some visual feedback that matched the reality of the command sequence.

**Comment by kjwoodsISIS** [comment]
As a temporary measure, how about creating an instrument script to issue the `C23,1` command?

**Comment by John-Holt-Tessella** [comment]
I think we can get away with doing nothing. It is a nice idea though, should I ask?

**Comment by kjwoodsISIS** [comment]
No, I don't think so.  I agree we can leave it alone for now.  If resetting the stress rig needs to be done before we update the IOC/OPI, then it should be a quick task to write and install the script.
